<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPUTweakWindow</class>
 <widget class="QWidget" name="GPUTweakWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPU Tweak</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="coreClockLabel">
     <property name="text">
      <string>Core Clock</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="coreClockInput">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QSlider" name="coreClockSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSlider" name="memoryClockSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="coreClockAutoCheckbox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="memoryClockInput">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="memoryClockLabel">
     <property name="text">
      <string>Memory Clock</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QSlider" name="fanSpeedSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="fanSpeedAutoCheckbox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="fanSpeedLabel">
     <property name="text">
      <string>Fan Speed</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="fanSpeedInput">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="memoryClockAutoCheckbox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="resetBtn">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="applyBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="ownRisksNoticeLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Using this feature may void your warranty ! Use at your own risks !</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="workInProgressLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;NOTE: This feature is a work in progress.&lt;/p&gt;&lt;p&gt;Only fan speed works if you enabled it in your xorg conf&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
